### PR TITLE
Fix: Invalid messages with too many plots

### PR DESCRIPTION
**Pull request by Chritzel**
This is the PR to Issue #20

### CODE REVIEW

**Comment by Chritzel** [comment]
I'm not quite sure what you mean by that - do you mean separating the print-statements into several lines?

File: plotter/Plotter.cpp [hunk 1]
```diff
@@ -167,15 +167,15 @@ void Plotter::Plot()
 {
     bool config = counter == 0;
 
-    Serial.print( "{\"" + TIME_KEY + "\":" ); Serial.print( millis() );
+    Serial.print( "{\""); Serial.print(TIME_KEY); Serial.print("\":" ); Serial.print( millis() );
```
Comment (devinaconley) [inline plotter/Plotter.cpp]: @Chritzel - for example this line would be:
```
Serial.print( "{\"" ); Serial.print( TIME_KEY ); Serial.print( "\":" ); Serial.print( millis() );
```

Comment (Chritzel) [reply plotter/Plotter.cpp]: Got it, added it.

**Comment by devinaconley** [comment]
this closes #21 

thanks @Chritzel !

**Pull request merged**